src/glob.zig

```diff
@@ -624,6 +784,18 @@ pub fn GlobWalker_(
             only_files: bool,
         ) !Maybe(void) {
             errdefer arena.deinit();
+            var cwd: []const u8 = undefined;
```

Comment:
Is this `cwd` supposed to be used with `initWithCwd` on line 788?

Reply:
I think this is leftover code, I removed the cwd stuff